curriculum/challenges/english/20-upcoming-python/learn-list-comprehension-by-building-a-case-converter-program/657f425dbab54e11993c80f0.md

```diff
@@ -0,0 +1,46 @@
+---
+id: 657f425dbab54e11993c80f0
+title: Step 15
+challengeType: 20
+dashedName: step-15
+---
+
+# --description--
+
+So far in this project you have not seen any usage of list comprehensions whatsoever. Like I said in the first step, list comprehensions are more concise way of constructing lists in Python.
+
+Begin the transition to comprehensions by commenting out all the lines of code inside the `convert_to_snake_case` function. Don't delete them as they'll helpful when implementing the logic using a comprehension.
```

Comment:
```suggestion
Begin the transition to list comprehensions by commenting out all the lines of code inside the `convert_to_snake_case` function. Don't delete them as they'll be helpful when implementing the logic using a comprehension.
```

Reply:
Either you ask to add `pass` after commenting out the function code or you don't use the `({test: () => })` format to test the code.